src/types/Widget.ts

```diff
@@ -160,6 +162,13 @@ export interface Style {
   inline_tile_size: string
   inline_tile_border_radius: string
   expanded_tile_border_radius: string
+  map_content_feed_filter_id: string
+  displayMode: string
+  isScrollZoomDisabled: boolean
```

Comment:
Can you confirm to the snakecase naming convention here? i.e. display_mode instead of displayMode.

Reply:
Dang legacy code